src/pdfkit.cs

```diff
@@ -2118,4 +2180,22 @@ interface PdfViewDelegate {
 		UIViewController ParentViewController { get; }
 	}
 
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	interface IPdfPageOverlayViewProvider {}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	[Protocol (Name = "PDFPageOverlayViewProvider")]
+	interface PdfPageOverlayViewProvider
+	{
+		[Abstract]
+		[Export ("pdfView:overlayViewForPage:")]
+		[return: NullAllowed]
+		UIView RequestPdfView (PdfView view, PdfPage page);
+
+		[Export ("pdfView:willDisplayOverlayView:forPage:")]
+		void WillDisplay (PdfView pdfView, UIView overlayView, PdfPage page);
```

Comment:
Also in this particular case the name sounds very generic, so I think it looks better if it's more explicit:

```suggestion
		void WillDisplayOverlayView (PdfView pdfView, UIView overlayView, PdfPage page);
```

One reason is that since this is a protocol, customers might end up implementing the corresponding interface in a class that does a lot of other things, and as such it's not always obvious that a method named "WillDisplay" is related to PdfPageOverlayViewProvider in any way.